packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -237,6 +236,72 @@ describe('FLE tests', () => {
     expect(plainMongoResult).to.not.include("phoneNumber: '+12874627836445'");
   });
 
+  context('6.0+', () => {
+    skipIfServerVersion(testServer, '< 6.0'); // FLE2 only available on 6.0+
+
+    it('drops fle2 collection with all helper collections when encryptedFields options are in listCollections', async() => {
+      const shell = TestShell.start({
+        args: ['--nodb'],
+        env: {
+          ...process.env,
+          MONGOSH_FLE2_SUPPORT: 'true'
+        },
+      });
+      const uri = JSON.stringify(await testServer.connectionString());
+
+      await shell.waitForPrompt();
+
+      await shell.executeLine('local = { key: BinData(0, "kh4Gv2N8qopZQMQYMEtww/AkPsIrXNmEMxTrs3tUoTQZbZu4msdRUaR8U5fXD7A7QXYHcEvuu4WctJLoT+NvvV3eeIg3MD+K8H9SR794m/safgRHdIfy6PD+rFpvmFbY") }');
+
+      await shell.executeLine(`keyMongo = Mongo(${uri}, { \
+        keyVaultNamespace: '${dbname}.keyVault', \
+        kmsProviders: { local } \
+      });`);
+
+      await shell.executeLine('keyVault = keyMongo.getKeyVault();');
+      await shell.executeLine('keyId = keyVault.createKey("local");');
+
+      await shell.executeLine(`encryptedFieldsMap = { \
+        '${dbname}.collfle2': { \
+          fields: [{ path: 'phoneNumber', keyId, bsonType: 'string' }] \
+        } \
+      };`);
+
+      await shell.executeLine(`autoMongo = Mongo(${uri}, { \
+        keyVaultNamespace: '${dbname}.keyVault', \
+        kmsProviders: { local }, \
+        encryptedFieldsMap \
+      });`);
+
+      // Drivers will create the auxilliary FLE2 collections only when explicitly creating collections
+      // via the createCollection() command.
+      await shell.executeLine(`autoMongo.getDB('${dbname}').createCollection('collfle2');`);
+      await shell.executeLine(`autoMongo.getDB('${dbname}').collfle2.insertOne({ \
+        phoneNumber: '+12874627836445' \
+      });`);
+
+      const autoMongoResult = await shell.executeLine(`autoMongo.getDB('${dbname}').collfle2.find()`);
+      expect(autoMongoResult).to.include("phoneNumber: '+12874627836445'");
```

Comment:
Since this is currently the only test for FLE2 support, do you want to also verify that this is returned as encrypted when using `plainMongo`, similar to the test above?